packages/UI/src/components/Checkbox/Checkbox.tsx

```diff
@@ -5,49 +5,73 @@ import CheckmarkImg from './../../assets/icons/Checkmark';
 const checkboxStyles = tv({
   slots: {
     rootStyle:
-      'flex h-6 w-6 items-center justify-center rounded-sm border-2 border-solid border-brand-alt-nebula-500 bg-grayscale-50 hover:outline hover:outline-4 hover:outline-offset-0 hover:outline-brand-alt-nebula-100 focus:outline focus:outline-2 focus:outline-offset-2 focus:outline-brand-alt-nebula-500 disabled:border-grayscale-500 disabled:bg-grayscale-200 disabled:hover:outline-none disabled:hover:outline-0 hover:disabled:outline-0 data-checked:bg-brand-alt-nebula-500 data-checked:disabled:bg-grayscale-200',
+      'flex h-6 w-6 items-center justify-center rounded-sm border-2 border-solid border-brand-alt-nebula-500 bg-grayscale-50 focus-within:outline focus-within:outline-2 focus-within:outline-offset-2 focus-within:outline-brand-alt-nebula-500 hover:outline hover:outline-4 hover:outline-offset-0 hover:outline-brand-alt-nebula-100 disabled:border-grayscale-500 disabled:bg-grayscale-200 disabled:hover:outline-none disabled:hover:outline-0 hover:disabled:outline-0 data-checked:bg-brand-alt-nebula-500 data-checked:disabled:bg-grayscale-200',
```

Comment:
The only thing I changed here is `focus:` to `focus-within:`, I actually made a mistake here as I should've changed it to `focus-visible:`.

[Read this](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#examples) which demonstrates the difference between `focus:` and `focus-visible:` and let me know what you think.